packages/ckb/src/utils/rgbpp.ts

```diff
@@ -18,7 +18,7 @@ export const genRgbppLockScript = (rgbppLockArgs: Hex, isMainnet: boolean) => {
 };
 
 export const genBtcTimeLockScript = (toLock: CKBComponents.Script, isMainnet: boolean) => {
-  const lockArgs = `${append0x(serializeScript(toLock))}${u32ToLe(BTC_JUMP_CONFIRMATION_BLOCKS)}`;
+  const lockArgs = `${append0x(serializeScript(toLock))}${u32ToLe(BTC_JUMP_CONFIRMATION_BLOCKS)}${RGBPP_TX_ID_PLACEHOLDER}`;
```

Comment:
related implementation in the rgbpp contracts:
https://github.com/ckb-cell/rgbpp/blob/8e9155a005a330c953bddbf2749b8c88da7500d0/contracts/rgbpp-lock/src/main.rs#L191-L204